awx_collection/plugins/doc_fragments/auth.py

```diff
@@ -10,52 +10,52 @@
 
 class ModuleDocFragment(object):
 
-    # Ansible Tower documentation fragment
+    # Ansible Controller documentation fragment
     DOCUMENTATION = r'''
 options:
-  tower_host:
+  controller_host:
     description:
-    - URL to your Tower or AWX instance.
-    - If value not set, will try environment variable C(TOWER_HOST) and then config files
```

Comment:
These variables matched what was coming from Tower for the "Ansible Tower" credential type. Are those also changing?

Reply:
This comment may have been on an outdated diff.

It looks like the environment variables are remaining the same for now. I will raise this point in an issue. Thanks for bringing it up.

We _have_ in the past aliased an environment variable for the config.

https://github.com/ansible/awx/blob/eec4f8dcc27428cd1aeb1c251d43052a3ea780e8/awxkit/awxkit/cli/format.py#L19

but that might have just been for the CLI.